lib/NetworkManager.js

```diff
@@ -341,6 +341,49 @@ class Request {
   }
 
   /**
+   * @param {!Object} response
```

Comment:
I'd feel safer if the options were typed here, considering we are getting a buffer.

Reply:
Done.